### PR TITLE
#78: Adding support for CronJobs

### PR DESCRIPTION
**Pull request by fbrnc**
Adding support for CronJobs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ scm-source.json
 htmlcov/
 .pytest_cache/
 .mypy_cache
+venv/
```
Comment (hjacobs) [inline .gitignore]: Please remove this change - the project does not use that directory (and Poetry uses Virtualenv under the hood with a different path)

**Comment by hjacobs** [comment]
Thanks!

File: .gitignore (hunk 1 above)
Comment (fbrnc) [reply .gitignore]: I'm relatively new to Python PyCharm created that dir for me. I don't think it hurts keeping this here but I understand if you want to keep that file clean.
However, I removed this line again.
Thanks for checking

Comment (hjacobs) [reply .gitignore]: As IDE-related files are highly depending on what people use (e.g. I don't use PyCharm), you can use a global gitignore for files created by PyCharm, see https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore

**Comment by runningman84** [comment]
is this released yet?

**Comment by hjacobs** [comment]
@runningman84 yes

**Pull request merged**